src/raftstore/store/snap.rs

```diff
@@ -49,6 +46,8 @@ use raftstore::store::metrics::{SNAPSHOT_BUILD_TIME_HISTOGRAM, SNAPSHOT_CF_KV_CO
                                 SNAPSHOT_CF_SIZE};
 use raftstore::store::peer_storage::JOB_STATUS_CANCELLING;
 
+use server::config::Config as ServerConfig;
```

Comment:
bad style to import server here

Reply:
I agree. In fact I think the config entry should be in `RaftStoreConfig` instead of `ServerConfig`. But there is arleady `max_write_bytes_per_sec` in `ServerConfig`. 

So could we move both them into `RaftStoreConfig`? It will be a break change because users must modify therir `tikv.toml` after the `TiKV` upgrade.